mitmproxy/addons/script.py

```diff
@@ -195,7 +201,9 @@ def configure(self, updated):
 
             ordered = []
             newscripts = []
-            for s in ctx.options.scripts:
+            for s in scripts:
+                if s[-2:] != "py":
+                    continue
```

Comment:
Any reason to hard-code and enforce only `.py` files?
I could imagine users want to write scripts (without extensions) and be able to load them into mitmproxy at the same time?